libs/ui-lib/lib/ocm/services/ClusterDetailsService.ts

```diff
@@ -39,12 +39,11 @@ const ClusterDetailsService = {
       baseDnsDomain: values.baseDnsDomain,
       cpuArchitecture: values.cpuArchitecture as ClusterCpuArchitecture,
       diskEncryption: DiskEncryptionService.getDiskEncryptionParams(values),
+      platform: {
+        type: values.platform,
+      },
     };
 
-    if (values.externalPartnerIntegrations) {
```

Comment:
`externalPartnerIntegrations` is not used anymore, needs more cleaning-up. In `getClusterDetailsInitialValues` etc